tools/autograd/derivatives.yaml

```diff
@@ -648,6 +648,11 @@
 - name: view(Tensor self, IntList size)
   self: grad.contiguous().view(self.sizes())
 
+- name: _s_where(Tensor condition, Tensor self, Tensor other)
```

Comment:
Shouldn't condition be a byte tensor? It's not differentiable

Reply:
It probably should be but it doesn't work right now; the issue is that gen_variable_type does function signature mapping based on the "type" in derivatives.yaml, not the "dynamic_type".  Let's look at fixing that in a different PR (or just combining native_functions.yaml and derivatives.yaml, since specifying that same function twice can only hurt you).

Reply:
https://github.com/pytorch/pytorch/issues/4279